chargeOclock/Source/DataSources/ClientsDataSource.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 
 class ClientsDataSource: NSObject, UITableViewDataSource {
 
-	let clients: [Client]
+	var clients: [Client]
```

Comment:
leave as a var, and add a `removeClient` method
(yes, it's a different name, because we'll be removing the client from a collection, the deletion happened elsewhere 😉 )